pandas/core/groupby/generic.py

```diff
@@ -1574,6 +1575,17 @@ def filter(self, func, dropna=True, *args, **kwargs):
 
         return self._apply_filter(indices, dropna)
 
+    def __getitem__(self, key):
+        # per GH 23566
+        if isinstance(key, tuple) and len(key) > 1:
+            warnings.warn(
+                "Indexing with multiple keys (implicitly converted to a tuple "
+                "of keys) will be deprecated, use a list instead.",
+                DeprecationWarning,
```

Comment:
use FutureWarning

why does this have to be len > 1?

Reply:
Because if it is exactly 1 then it actually becomes a GroupBySeries for which this is valid syntax.

Reply:
ok, can you add a comment to that effect